bootstrap/manifest_template.json

```diff
@@ -515,6 +515,11 @@
     "url": "http://status-web.discoverd"
   },
```

Comment:
I think we should drop this step. If the status endpoint is not green, it will never pass this point due to the 500. It's probably worth manually testing this out by breaking a service or two and confirming that the output is correct.